client/src/app/models/task.ts

```diff
@@ -0,0 +1,21 @@
+export interface Task {
+    _id: string;
+    title: string;
+    description: string;
+    createdAt: Date;
+    updatedAt: Date;
+    createdBy: string;
+    tasks:[string];
+}
+
+export interface TaskUpdate{
+    _id:string;
+    title: string;
+    description:string;
+}
+export interface TaskAdd{
+    title: string;
+    description:string;
+    createdBy:string;
+    parentId:string;
```

Comment:
parentId can be empty
**parentId:string** -> **parentId?:string**

Reply:
Now parentId can be null also